From b76d56be0a0d43a9cbe5f403f92b6492a622f023 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 20 Aug 2018 11:50:20 +0300
Subject: [PATCH] Makefile: add pointer for latest gaudi/dali-release

Add a single name pointer (tag) to the latest release.
This way other systems (e.g. jenkins) can refer to the same name and
get each time the new release.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index c2de81f..655fe12 100644
--- a/Makefile
+++ b/Makefile
@@ -63,12 +63,15 @@
 
 #TODO: add btl once it is separated from br-ext
 RELEASE_REPOS := buildroot-external preboot u-boot linux armcp tbot
+RELEASE_TAG_POINTER := latest-embedded-release
 
 .PHONY: gaudi-release dali-release
 gaudi-release dali-release: hl-check-release
 	$(eval HL_FULL_VER=hl-$(patsubst %-release,%,$@)-$(HL_RELEASE_VER))
 	@repo forall $(RELEASE_REPOS) -c \
 		git tag -afm "Release Habanalabs $(HL_FULL_VER)" $(HL_FULL_VER)
+	@repo forall $(RELEASE_REPOS) -c \
+		git tag -f $(RELEASE_TAG_POINTER)
 	@repo forall $(RELEASE_REPOS) -c git push origin $(HL_FULL_VER)
 	@echo "Released: $(HL_FULL_VER)"
 
